qcengine/programs/nwchem/germinate.py

```diff
@@ -63,4 +74,15 @@ def muster_modelchem(method: str, derint: int, use_tce: bool) -> Tuple[str, Dict
         else:
             mdccmd = f"task ccsd(t) {runtyp}\n\n"
 
+    elif method in ['sodft', 'direct_mp2', 'rimp2', 'mscf', 'selci', 'md', 'pspw', 'band']:
+        raise ValueError(f"Method \"{method}\" not yet supported by QCEngine")
```

Comment:
"mscf" --> "mcscf"?

perhaps better to whitelist dft methods and leave anything else to raise "not yet supported"?

Reply:
@wardlt does this need a fix?

Reply:
The typo was fixed and I added a whitelist, as suggested

Reply:
Ah, this is outdated, wish they continued to autoclose comments after edits.